cli/src/utils/initTestEnv.ts

```diff
@@ -92,32 +102,42 @@ export async function start_test_validator({
   const command = "solana-test-validator";
   const LIMIT_LEDGER_SIZE = "500000000";
   const BASE_PATH = "../../bin/";
-  type Program = { id: string; name?: string; path?: string };
+  type Program = { id: string; name?: string; tag?: string; path?: string };
   const programs: Program[] = [
-    { id: "noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV", name: "spl_noop.so" },
+    {
+      id: "noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV",
+      name: "spl_noop.so",
+      tag: SPL_NOOP_PROGRAM_TAG,
```

Comment:
It looks to me as if these tags are not used yet, but still PROGRAM_TAG.
see: https://github.com/Lightprotocol/light-protocol/pull/347/files#diff-ae098510c28e4ea810a9c0f3b0a070fccdd83301a3e2d7c424896ca2fb2474c2R167
Is this on purpose?